src/services/utilities.ts

```diff
@@ -1357,6 +1357,13 @@ namespace ts {
             !bindingElement.propertyName;
     }
 
+    export function getObjectBindingElementWithoutPropertyNameOrUndefined(symbol: Symbol): BindingElement & { name: Identifier } | undefined {

```

Comment:
>BindingElement & { name: Identifier } [](start = 91, length = 37)

`ObjectBindingElementWithoutPropertyName`?

Reply:
Removed this utility function.